src/contracts/pods/EigenPod.sol

```diff
@@ -124,6 +124,11 @@ contract EigenPod is IEigenPod, Initializable, ReentrancyGuardUpgradeable, Eigen
         _;
     }
 
+    modifier partialWithdrawalProofSwitchOff() {
```

Comment:
After you make the `require` change I mentioned in another comment, this modifier should be removed

Reply:
yup already removed in favor of an explicit check